### PR TITLE
[Doc] PR for Document upgrade order of HA primary and standby nodes

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-5166

### CODE REVIEW

**Comment by lizayugabyte** [comment]
@sb-yb Could you, please, review this doc PR ?

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md [hunk 1]
```diff
@@ -93,6 +93,16 @@ The restore take a few seconds, after which expect to be logged out.
 
 You should be able to see that all of the data has been restored into the instance, including universes, users, metrics, alerts, task history, cloud providers, and so on.
 
+## Upgrade instances
+
+All instances involved in high availability should be of the same YugabyteDB Anywhere version. If the versions are different, an attempt to promote a standby instance using a YugabyteDB Anywhere backup from an active instance will result in errors.
```
Comment (sb-yb) [inline docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:98]: may result in errors.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md [hunk 2]
```diff
@@ -93,6 +93,16 @@ The restore take a few seconds, after which expect to be logged out.
...
+
+Even though you can perform an upgrade of all YugabyteDB Anywhere instances simultaneously and there are no explicit ordering requirements regarding upgrades of active and standby instances, it is recommended to follow these guidelines: 
+
+- Start an upgrade with an active instance.
+- After the active instance has been upgraded, ensure that YugabyteDB Anywhere is reachable by logging in and checking various pages.
+- Proceed with upgrading standby instances, possibly in parallel.
```
Comment (sb-yb) [inline docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:104]: you can remove ", possibly in parallel" 
I know I had suggested that in slack but on the second read it looks bit confusing. Some users may take it to mean that it should be done in parallel with active instance (step 1).

**Review by sb-yb — APPROVED** [review]
Two suggestions.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:98]: Fixed

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:104]: Fixed

**Pull request merged**